Update: flutter getting started to use amplify studio. (#3973)
aws-amplify/docs: src/fragments/start/getting-started/flutter/add-api.mdx

    ![Create app backend](/images/lib/getting-started/flutter/connect-to-cloud-create-app-backend.png)

> 4. The on-screen text should walk you through the deployment process and, when the deployment status reads **Deployment completed**, click **Launch Studio**.

Reviewer: This still reads a bit off to me: 
- usage of `should` implies that there is potential that the on-screen text will not walk the reader through the deployment process
- the comma after `Deployment created` is extraneous
- imo this is two sentences, the first being informational, and the second action oriented

```suggestion
4. The on-screen text will walk you through the deployment process. Once the deployment status reads **Deployment completed** click **Launch Studio**.
```

> 4. The on-screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.

Reviewer: I am a bit confused by the phrasing `The on-screen text should walk you through the deployment progress`, should this be `The on-screen text should walk you through the deployment process`?

Author: Thanks Caleb, I have updated it.

4. The on-screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.

>     ![Open admin UI](/images/lib/getting-started/flutter/connect-to-cloud-open-admin-ui.png)

Reviewer: Is the corresponding file still used? If not can it be deleted?

Author: Yes Caleb. It is not being used. I have removed it.

>     ![Open admin UI](/images/lib/getting-started/flutter/connect-to-cloud-open-amplify-studio.png)

Reviewer 2: This could be important to screen readers maybe?
```suggestion
    ![Open Amplify Studio](/images/lib/getting-started/flutter/connect-to-cloud-open-amplify-studio.png)
```
